python/ray/data/datasource/parquet_datasource.py

```diff
@@ -278,6 +278,11 @@ def get_read_tasks(self, parallelism: int) -> List[ReadTask]:
                 pieces=pieces,
                 prefetched_metadata=metadata,
             )
+            # If there is a filter operation, reset the calculated row count,
+            # since the resulting row count is unknown.
+            if self._reader_args.get("filter") is not None:
+                meta.num_rows = None
```

Comment:
do we also need to reset `meta.size_bytes` here?

Reply:
let's not reset `size_bytes` to None, because that will make streaming executor throw exception.